Dockerfile

```diff
@@ -36,13 +36,17 @@ RUN set -ex; \
       libpng-dev \
       libfreetype6-dev \
       libzip-dev \
+      libldap2-dev \
     ; \
     \
   docker-php-ext-configure gd \
         --with-jpeg-dir=/usr \
         --with-png-dir=/usr \
     --with-freetype-dir=/usr \
   ; \
+  docker-php-ext-configure ldap \
+        --with-libdir=lib/x86_64-linux-gnu/ \
```

Comment:
what is the default `libdir` ? Is this necessary?

Reply:
Point to `/usr/lib/x86_64-linux-gnu/` to access `.so` files installed with `apt-get install libldap2-dev`